platform/linux_platform.go

```diff
@@ -569,6 +573,49 @@ func (p linux) SetupEphemeralDiskWithPath(realPath string) error {
 		return bosherr.WrapError(err, "Mounting data partition")
 	}
 
+	if p.options.ScrubEphemeralDisk {
+		agentVersionFilePath := path.Join(p.dirProvider.DataDir(), ".bosh", "agent_version")
```

Comment:
I think it is better to keep agent_version in a hidden dir `/var/vcap/data/.bosh/agent_version`  as it is only generated and used when ScrubEphemeralDisk is true. What do you think?